Angular/src/app/app.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component } from '@angular/core';
+import { rrulestr, RRule } from 'rrule';
+import { CloseButtonOptions } from './interfaces';
+import { appointments, Appointment } from './data/appointments'
+import { AppointmentAddingEvent } from "devextreme/ui/scheduler";
```

Comment:
Let's replace double quotes with single quotes:

```suggestion
import { AppointmentAddingEvent } from 'devextreme/ui/scheduler';
```